engine/app/components/citizens_advice_components/targeted_content.html.haml

```diff
@@ -1,9 +1,10 @@
 .cads-targeted-content.js-cads-targeted-content{ attributes }
-  - haml_tag("h#{heading_level}", heading_attributes) do
-    .cads-targeted-content__title-text
-      - if adviser?
-        .cads-targeted-content__badge
-          = render(CitizensAdviceComponents::Badge.new(type: :adviser))
-      = title
+  .heading-wrapper
```

Comment:
Where does this `.heading-wrapper` class come from? Isn't this the class added dynamically by references?